examples/ss/ss_other.go

```diff
@@ -0,0 +1,11 @@
+// +build !linux
+
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("This command is only available on Linux")
```

Comment:
Isn't  this command is NOT available on linux?

Reply:
`ss.go` contains a linux-only implementation, I created this one to avoid these errors on other platforms:
```
can't load package: package github.com/elastic/gosigar/examples/ss: build constraints exclude all Go files in /Users/travis/gopath/src/github.com/elastic/gosigar/examples/ss
/Users/travis/.travis/job_stages: line 78: ./examples/ss/ss: No such file or directory
```